account_multicurrency_revaluation/model/account.py

```diff
@@ -54,83 +78,109 @@ def _onchange_user_type_id(self):
             if rec.user_type_id.id in revaluation_accounts:
                 rec.currency_revaluation = True
 
-    def _revaluation_query(self, revaluation_date):
-
+    def _revaluation_query(self, start_date, revaluation_date):
```

Comment:
IMHO we can enhance compatibility by using different method signature.
```suggestion
    def _revaluation_query(self, revaluation_date, start_date=None):
```